### DISCUSSION TITLE
Move serialize/deserialize functions from wagmi to viem

### DISCUSSION THREAD

**Discussion by rkalis** [wevm/viem#1160] [Idea / Feature Request]
The bigint-enabled lossless serialization that is in wagmi (see https://wagmi.sh/core/faq#bigint-serialization) seems like it is aligned with viem as it is not specific to client-side web usage, which is the main use case of wagmi.

**Comment by joaofranciscoguarda** [comment]
I agree, that's not a wagmi only logic, I'll copy the code from wagmi in my repo because it's not in viem package, and I dont need wagmi for now
